admin/src/pages/View/components/NavigationManager/index.tsx

```diff
@@ -0,0 +1,244 @@
+// @ts-ignore
+import { Flex } from "@strapi/design-system/Flex";
+// @ts-ignore
+import { Loader } from "@strapi/design-system/Loader";
+import {
+  ModalBody,
+  ModalFooter,
+  ModalHeader,
+  ModalLayout,
+  // @ts-ignore
+} from "@strapi/design-system/ModalLayout";
+import React, { useCallback, useEffect, useMemo, useState } from "react";
+import { useIntl } from "react-intl";
+import useDataManager from "../../../../hooks/useDataManager";
+import { getMessage } from "../../../../utils";
+import { Create, createFooterActions } from "./Create";
+import { Delete, deleteFooterActions } from "./Delete";
+import { Edit, editFooterActions } from "./Edit";
+import { ErrorView } from "./Error";
+import { List, listFooterActions } from "./List";
+import { FooterActionsFactory, Navigation, SetState, State } from "./types";
+
+interface Props {
+  initialState: State;
+  onClose?: () => void;
+  isOpened?: boolean;
+}
+
+export const NavigationManager = ({
+  initialState,
+  onClose,
+  isOpened,
+}: Props) => {
+  const { formatMessage } = useIntl();
+  const [state, setState] = useState(initialState);
+  const {
+    items = [],
+    handleNavigationsDeletion,
+    handleSubmitNavigation,
+    hardReset,
+  } = useDataManager();
+  const navigations = useMemo(
+    () =>
+      items
+        .map((x: Navigation) => x)
+        .sort((a: Navigation, b: Navigation) => Number(a.id) - Number(b.id)),
```

Comment:
I think we could use lodash
```suggestion
    () => _.sortBy(items, 'id')
```